### PR TITLE
Add flexibility to GetStation utterances

### LINKED ISSUE #50: Make GetStation Utterance more flexible
"ask metro transit when the next train is leaving ballston" causes a GetStation intent with undefined station slot

### PR DESCRIPTION
**Pull request by r33beers**
Fixes #50

### CODE REVIEW

**Comment by r33beers** [comment]
Adding `?w=1` to the PR url will help you review this one, since I changed white space.

File: speechAssets/SampleUtterances.txt [hunk 1]
```diff
@@ -1,8 +1,12 @@
 GetHelp help
-GetStation	when is the next train arriving at {pentagon city|station}
-GetStation	when is the next train arriving at {ballston|station}
-GetDestinationStation	{wiehle-reston east|destinationStation}
-GetDestinationStation	{new carrollton|destinationStation}
-GetServiceAdvisories    {Advisories}
-GetServiceAdvisories    are there any {Advisories}
-GetServiceAdvisories    list {Advisories}
+GetStation  departures at {ballston|station}
```
Comment (pmyers88) [inline speechAssets/SampleUtterances.txt]: Isn't it more natural to say departures from?

Comment (r33beers) [inline speechAssets/SampleUtterances.txt]: Yes. Let me fix that. I had 'from' there before...

Comment (r33beers) [inline speechAssets/SampleUtterances.txt]: FYI, amazon will handle minor word changes like that though.

**Pull request merged**